src/component/mxgraph/MxGraphRenderer.ts

```diff
@@ -70,7 +70,7 @@ export default class MxGraphRenderer {
       const bounds = shape.bounds;
       let labelBounds = shape.label?.bounds;
       // pool/lane label bounds are not managed for now (use hard coded values)
-      labelBounds = ShapeUtil.isPoolOrLane(bpmnElement.kind) ? undefined : labelBounds;
+      labelBounds = ShapeUtil.isPool(bpmnElement.kind) ? undefined : labelBounds;
```

Comment:
:warning: not related to this PR, we are not managing lane labels here

Reply:
ok, will be moved